router.js

```diff
@@ -0,0 +1,85 @@
+const router = require("express").Router();
+const dbHelper = require("./data/dbHelper");
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+const env = require("dotenv");
+env.config();
+
+router.get("/", (req, res) => {
+  res.status(200).json({ message: "The router at /api is working" });
+});
+
+router.get("/users", (req, res) => {
+  dbHelper
+    .getUsers()
+    .then(users => {
+      res.status(200).json(users);
+    })
+    .catch(err => {
+      res.status(500).json(err.message);
+    });
+});
+
+router.post("/register", (req, res) => {
```

Comment:
note here you are requesting the user send the info on the headers but are using a post route. POST is the common use for a register and login so i would keep that part and modify the req to look at the body instead of the headers for this info